data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/jsonview/Student.java

```diff
@@ -16,14 +18,18 @@ public class Student {
     @GeneratedValue(GeneratedValue.Type.IDENTITY)
     private Long id;
     private String name;
+    @Nullable
     private Double averageGrade;
 
+
+    @Nullable
     private LocalDateTime startDateTime;
 
     private boolean active;
 
-    @JoinTable(name = "TBL_STUDENT_CLASSES")
-    @Relation(Relation.Kind.MANY_TO_MANY)
+    @JoinTable(name = "TBL_STUDENT_CLASSES", joinColumns = {@JoinColumn(name = "student_id")},
```

Comment:
Noticed one issue with this student schema and `TBL_STUDENT_CLASSES` many-to-many table. It gets created on the fly but there is also an entity representing it with additional fields. On the fly creates only fks and we should be able to prevent creation. One way I found is adding mappedBy